Tests/Marketplace/marketplace_constants.py

```diff
@@ -186,6 +186,7 @@ class Metadata(object):
     URL = 'url'
     MARKETPLACES = 'marketplaces'
     DISABLE_MONTHLY = 'disableMonthly'
+    METADATA_FILE_REGEX_GET_VERSION = r'metadata\-([\d\.]+)\.json'
```

Comment:
The Metadata class is more for metadata properties, please define this variable as a separate constant var from the metadata